plugins/pulp_rpm/plugins/distributors/yum/metadata/updateinfo.py

```diff
@@ -133,13 +133,16 @@ def _get_package_checksum_tuple(self, package):
 
         return package_checksum_tuple
 
-    def add_unit_metadata(self, item):
+    def add_unit_metadata(self, item, filtered_pkglist):
```

Comment:
@seandst , Could you elaborate, what is the case when `filtered_pkglist` can be `None`?